testapi.pm

```diff
@@ -720,6 +721,84 @@ sub assert_shutdown(;$) {
     die "Machine didn't shut down!";
 }
 
+=head2 parse_junit_log
+
+parse_junit_log("report.xml")
+
+Upload log file from SUT (calls upload_logs internally). The uploaded
+file is then parsed as jUnit format and extra test results are created from it.
+
+=cut
+
+sub parse_junit_log {
+    my ($file) = @_;
+
+    upload_logs($file);
+
+    $file =~ s/.*\///;
```

Comment:
we use basename in this file already - and you should too because it makes quite more obvious what's going on.